### DISCUSSION TITLE
Optimization an max camera number

### DISCUSSION THREAD

**Discussion by matteopizzoli** [blakeblackshear/frigate#903] [Ask A Question]
Hi,
i'm running frigate on HA like add on (virtualbox) and i'm migrating in docker (same computer) for use coral on minipcie.
Hp 8200 sff 8Gb 256 ssd disk.
in the VirtualBox i see inference at 130ms in Docker 18ms.
the question is:
inference time increase with number/resolution/FPS or is a parameter that is not influenced by this?
what parameter and relative value i have to see to reach the maximum (and usable) number/resolution/fps of cameras added?
 thanks 
Matteo

**Comment by blakeblackshear** [comment]
Inference time is not impacted by any of those. Lower inference times are best. Intel NUCs typically see under 10ms inference times.

**Reply by matteopizzoli** [reply]
intel NUC without coral stay under 10 ms??? it's so fast
i woud use more than one pcie coral (it's quite cheap, 21 euros..) but ufortunately i have only one minipcie expansion, by the way i've added 6 camera (720 x 576 detect, full hd clips 4fps) in docker and 2 in VM and everityng work fine (i've not tried with a lot a movement, we are in lockDown :-) )
